src/Hedgehog/Range.fs

```diff
@@ -109,52 +109,52 @@ module Range =
                 min y (max x n)
 
         /// Scale an integral linearly with the size parameter.
-        let inline scaleLinear (sz0 : Size) (z0 : 'a) (n0 : 'a) : 'a =
+        let inline scaleLinear (sz0 : Size) (o0 : 'a) (n0 : 'a) : 'a =
             let sz =
                 max 0 (min 99 sz0)
 
-            let z =
-                toBigInt z0
+            let o =
+                toBigInt o0
 
             let n =
                 toBigInt n0
 
             let diff =
-                ((n - z) * bigint sz) / (bigint 99)
+                ((n - o) * bigint sz) / (bigint 99)
 
-            fromBigInt (z + diff)
+            fromBigInt (o + diff)
 
         /// Scale an integral exponentially with the size parameter.
-        let inline scaleExponential (lo : 'a) (hi : 'a) (sz0 : Size) (z0 : 'a) (n0 : 'a) : 'a =
+        let inline scaleExponential (lo : 'a) (hi : 'a) (sz0 : Size) (o0 : 'a) (n0 : 'a) : 'a =
```

Comment:
I think these should be consistently named with everything else (`lowerBound`, `upperBound`, etc.)

Also, I never liked that the parameters had numerals in their names (`sz0`, `z0`, etc.) If we must keep this naming, then the parameters should be named simply, and names with numerals should only be internally visible.